### PR TITLE
feat: LT-2061 - Export Licence for Particular Type (Cryptographic)

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: applications/libraries/goodstype_category_helpers.py [hunk 1]
```diff
@@ -5,12 +5,37 @@
...
 
 
+def set_destinations_for_uk_continental_shelf_application(application):
+    country = Country.include_special_countries.get(id="UKCS")
+    CountryOnApplication(country=country, application=application).save()
+
+
+def set_goods_and_countries_for_open_dealer_application(application):
+    _add_goodstypes_from_csv("DEALER", application)
+    for country in Country.objects.filter(is_eu=1).exclude(id="GB"):
```
Comment (jfaracik) [inline applications/libraries/goodstype_category_helpers.py:15]: `is_eu=1` can that just be `is_eu=True` ;P

Comment (cstobart) [reply applications/libraries/goodstype_category_helpers.py]: done 👍

File: applications/libraries/goodstype_category_helpers.py [hunk 2]
```diff
@@ -5,12 +5,37 @@
...
 
 
+def set_destinations_for_uk_continental_shelf_application(application):
+    country = Country.include_special_countries.get(id="UKCS")
+    CountryOnApplication(country=country, application=application).save()
+
+
+def set_goods_and_countries_for_open_dealer_application(application):
+    _add_goodstypes_from_csv("DEALER", application)
+    for country in Country.objects.filter(is_eu=True).exclude(id="GB"):
```
Comment (a-bek) [inline applications/libraries/goodstype_category_helpers.py:15]: https://docs.djangoproject.com/en/3.0/ref/models/querysets/#bulk-create

Also use in other areas where you do simple loop creates.

File: test_helpers/clients.py [hunk 3]
```diff
@@ -141,18 +141,37 @@ def setUp(self):
         [
             ControlListEntry.create(clc, "Description", None, False)
             for clc in [
+                "ML6b2",
```
Comment (a-bek) [inline test_helpers/clients.py:144]: Bulk here too?

File: applications/libraries/goodstype_category_helpers.py (hunk 2 above)
Comment (cstobart) [reply applications/libraries/goodstype_category_helpers.py]: done 👍

File: test_helpers/clients.py (hunk 3 above)
Comment (cstobart) [reply test_helpers/clients.py]: done 👍

**Pull request merged**